src/main/java/Duke.java

```diff
@@ -1,10 +1,190 @@
+import java.util.Scanner;
+//import java.util.Arrays;
+
 public class Duke {
+
+    private static final Scanner input = new Scanner(System.in);
+
     public static void main(String[] args) {
```

Comment:
Applying SLAP would make main function more readable.
An example is you could refactor the switch into a method call executeCommand().